.github/workflows/publish-to-dockerhub.yml

```
@@ -29,15 +28,22 @@ jobs:
       - name: Get tags
         run: git fetch --tags origin
 
-      - name: Login to DockerHub
+      - name: Set GitHub account and container registry repo variables
+        run: |
+          echo "GITHUB_ACCOUNT=${GITHUB_REPOSITORY%%/*}" >> $GITHUB_ENV
+          echo "GH_CONTAINER_REGISTRY_REPO=${GITHUB_REPOSITORY#*/}" >> $GITHUB_ENV
+
+      - name: Login to GitHub Container Registry
         uses: docker/login-action@v3
         with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_TOKEN }}
+          registry: ghcr.io
+          username: ${{ env.GITHUB_ACCOUNT }}
+          password: ${{ secrets.GH_PAT}}
```

Maybe we could replace this with GH_TOKEN too which is currently collected from the developer as part of the "setup environment" script 🤔 

https://github.com/opencrvs/opencrvs-countryconfig/blob/develop/infrastructure/environments/setup-environment.ts#L271-L282

```suggestion
          password: ${{ secrets.GH_TOKEN}}
```